doc/manage-duniverse.mld

```diff
@@ -0,0 +1,116 @@
+{1 Considerations}
+
+When using opam-monorepo the question arises: how to manage it in comparison to
+the regular opam workflow? Compared to regular opam usage using opam-monorepo
+brings lockfiles and a duniverse folder into the repository. There are multiple
+ways to handle this, with multiple advantages and disadvantages. This document
+explains the considerations behind dealing with each of those, along with our
+recommendations.
+
+{1 Methods}
+
+We assume the opam file is committed to the repository. Running [opam monorepo
+lock] will create a [.locked] opam file in addition to the original opam file.
+Calling [opam monorepo pull] will create a [duniverse] folder into which it
+will unpack the sources of the packages that are mentioned in the lockfile.
+
+{2 Not locked}
+
+While this is not a [opam-monorepo] workflow per se, projects that do not use
+[opam-monorepo] can still be used with [opam-monorepo]. As such every
+contributor is responsible for creating and maintianing their own lockfile and
+duniverse.
```

Comment:
Do you mean calling `opam lock` or sourcing all your dependencies manually (with a script that generates https://github.com/janestreet/universe)?

Reply:
btw, the MirageOS tool is doing `opam-monorepo lock && opam-monorepo pull`, which is a perfectly valid use-case for the desired workflow (set-up a dune workspace to perform cross-compilation). Would also be nice to not dismiss this use-case if possible.

Reply:
I meant the latter. You're right, I haven't thought of this usecase, I will add it.